Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.5
```

Comment:
Is this an all-new file? Was it not needed before?

How is this file created?

Reply:
This is the actual package created by our internal Jenkins build. This package is used by users to include in their projects.

Reply:
How does it get pushed into the repo?

Reply:
By our Jenkins job.

Reply:
Got it. Do we need to merge it into `master`? What is the purpose of the `master` branch in this repo?

Reply:
(I guess that's where the docs live?)

Reply:
Master is just where the latest package/docs live. We will want to merge this branch into master then create a GitHub Release that automatically creates a version tag. While not releasing, we should have the `master` branch be our only branch.

Reply:
It might be good for us to rename that branch to `main` before we go public with this package.

Reply:
I took the liberty of renaming it

Reply:
Thanks, @alonzno. 

@spencerairtime Can we create the GitHub release from the release branch? Just curious.

The branching scheme we're using here is a little different from what we do in other projects. It's OK to be different, but using the same naming scheme for a different strategy is potentially a little confusing.

Once we get the docs moved onto the web site, will there be any need for a PR at all? Can Jenkins push directly to the main branch?

Reply:
We could create a GitHub release from this branch, but I don't think we should. We'd want `main` to always have the latest and have releases target `main` commits as well.

In theory, no one should notice nor use these release branches. They should always be using tags. But I did like this naming scheme because it looks nice in the URL.

Once we have the docs moved, I guess there is no need for PR's. Although we should still have a README.md here because that is utilized by XCode when you include our package.

Reply:
Can we copy the README.md over from the Asilomar repo so that it gets versioned along with the source code?

Reply:
Yeah that would work. That means our internal Asilomar README.md would look the same as our public facing README.md.